### PR TITLE
parser script to gather polarion details from test suites

### PR DESCRIPTION
**Pull request by sunilkumarn417**
- Parser script to collect the Polarion information from every test case in suites directory.

**Usage:** `python retrieve_polarion_ids.py ~/Sunil/recipe/cephci/suites --exclude luminous,jewel,integrations `               

**Sample output:** http://magna002.ceph.redhat.com/ceph-qe-logs/Sunil/polarion_data.yaml

Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

### CODE REVIEW

File: utility/retrieve_polarion_ids.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
+usage = """Retrieve Polarion Ids from test suites.
```
Comment (psathyan) [inline utility/retrieve_polarion_ids.py:1]: Could we reword the statement? I believe we are determining if the test cases implemented are inline with the stated guidelines of having 1 to 1 mapping with a Test case.

File: utility/retrieve_polarion_ids.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    return {
+        "multiple_ids": multiple_ids,
+        "missing_ids": missing_ids,
+        "good": good,
+        "total": _total,
+    }
+
+
+if __name__ == "__main__":
+    logging.basicConfig(
```
Comment (psathyan) [inline utility/retrieve_polarion_ids.py:94]: Do we need `logging` as we are mainly using `print` statements? We could move all print statements to `log`

Comment (sunilkumarn417) [reply utility/retrieve_polarion_ids.py]: done

File: utility/retrieve_polarion_ids.py (hunk 1 above)
Comment (sunilkumarn417) [reply utility/retrieve_polarion_ids.py]: Done

**Review by psathyan — APPROVED** [review]

**Pull request merged**